Header header
float64 gain